libraries/SITL/SIM_AirSim.h

```diff
@@ -0,0 +1,86 @@
+/* 
+	Simulator connector for Airsim: https://github.com/Microsoft/AirSim
+*/
+
+#pragma once
+
+#include <AP_HAL/utility/Socket.h>
+#include "SIM_Aircraft.h"
+
+namespace SITL {
+
+/* 
+	Airsim Simulator
+*/
+
+class AirSim : public Aircraft {
+public:
+	AirSim(const char *home_str, const char *frame_str);
+
+	/* update model by one time step */
+	void update(const struct sitl_input &input) override;
+
+	/* static object creator */
+    static Aircraft *create(const char *home_str, const char *frame_str) {
+        return new AirSim(home_str, frame_str);
+    }
+
+private:
+	/*
+		packet sent to Ardupilot
+		Taken from Airsim ArduCopter Solo connector
+	*/
+	// No idea why it's 11, HIL_ACTUATOR_CONTROLS message takes 16 servo values
+	// This can possibly be changed in Airsim
+	// Actually, AirSim is only taking in 8 PWM values, need to look into this
+	// It's supports both HIL_CONTROLS & HIL_ACTUATOR_CONTROLS message but uses only 8 values
+	static const int kArduCopterRotorControlCount = 11;
+
+	struct servo_packet {
+		// ArduPilot Solo rotor control datagram format
```

Comment:
We should probably not reference Solo in here unless stuff really is Solo-specific :-)

Reply:
Well, yes, this entire connector is following the old Solo format since, in Airsim, they have added support for that

Reply:
Removed